### PR TITLE
Update README to v3

### PR DESCRIPTION
**Pull request by vsafonkin**
Updating the examples in README to the action version `v3`

Issue: #203

### CODE REVIEW

**Review by marko-zivic-93 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ This action sets up a go environment for use in actions by:
...
 - registering problem matchers for error output
 
-# V2
+# V3
 
-The V2 offers:
+The V3 offers:
 - Adds GOBIN to the PATH
 - Proxy Support
 - `stable` input
```
Comment (brcrista) [inline README.md:19]: This has actually been removed with V3: https://github.com/actions/setup-go/releases/tag/v3.0.0

Comment (marko-zivic-93) [reply README.md:19]: Thank you Brian for pointing this out :)

Comment (vsafonkin) [reply README.md]: Fixed, thanks!

**Review by brcrista — APPROVED** [review]
thanks!

**Pull request merged**